File: crates/rhizome-treesitter/tests/fixtures/sample.py
"""Sample Python module for testing."""

import os
from pathlib import Path


class Config:
    """Configuration class."""

    def __init__(self, name: str, value: int):
        self.name = name
        self.value = value

    def get_value(self) -> int:
        """Return the value."""
        return self.value


def process(config: Config) -> str:
    """Process a config."""
    return f"{config.name}: {config.value}"


MAX_SIZE = 1024

class Status:
    ACTIVE = "active"
    INACTIVE = "inactive"

File: crates/rhizome-mcp/tests/fixtures/sample.rs
/// A sample struct for testing
#[derive(Debug)]
pub struct Config {
    pub name: String,
    pub value: i32,
}

impl Config {
    /// Creates a new Config
    pub fn new(name: String, value: i32) -> Self {
        Config { name, value }
    }

    pub fn value(&self) -> i32 {
        self.value
    }
}

/// A standalone function
pub fn process(config: &Config) -> String {
    format!("{}: {}", config.name, config.value)
}

const MAX_SIZE: usize = 1024;

use std::collections::HashMap;
use std::path::PathBuf;

pub enum Status {
    Active,
    Inactive,
}

pub trait Processor {
    fn process(&self) -> Result<(), String>;
}
